recipe/meta.yaml

```diff
@@ -43,61 +44,150 @@ requirements:
     - xorg-libxext  # [linux]
     - xorg-libxrender  # [linux]
     - zlib
-  run:
-    - xorg-libice  # [linux]
-    - xorg-libsm  # [linux]
-    - xorg-libx11  # [linux]
-    - xorg-libxext  # [linux]
-    - xorg-libxrender  # [linux]
-    # hmaarrfk: 2022/08
-    # While the dependencies below should be captured
-    # by run_exports, run_exports doesn't necessary
-    # bring in compile time depenencies
-    - zlib
 
-test:
-  requires:
-    - glib  # needed for the pkg-config invocations below
-    - pkg-config
+outputs:
+
+  # Everything needed to build against libcairo
+  - name: cairo
```

Comment:
Because the "everything minus static libs" package is named "cairo" existing downstream packages should not break. Newly built packages will only recieve exports of the numbered shared libraries. (I believe cairo-trace is a dev package that does not need to be exported).